common/authorization/default_token_key_provider.go

```diff
@@ -110,6 +110,10 @@ func (a *defaultTokenKeyProvider) EcdsaKey(alg string, kid string) (*ecdsa.Publi
 	return key, nil
 }
 
+func (a *defaultTokenKeyProvider) SupportedMethods() []string {
+	return []string{"RS256", "ES256"}
```

Comment:
Just curious is it necessary for this to be uppercase now? Also can we make these a const, it seems to be reused throughout this code.

Reply:
Good point. I replaced them with references to jwt.SigningMethod* now.